R/get_meta.R

```diff
@@ -9,21 +9,19 @@
 #' @return tbl_df
 #'
 #' @examples
-#'
+#' \dontrun{
```

Comment:
is there a reason some examples have been removed, and some set to don't run? Should we be consistent across files?